### DISCUSSION TITLE
Proxy and Self-Signed Certs / CA Pathing

### DISCUSSION THREAD

**Discussion by lostpolaris** [CrowdStrike/falconpy#908] [Q&A] [answered]
Hey ya'll, 

Was just wondering if we can pass in proxies to this library using Environment Variables and also, if we're behind a corporate proxies that does SSL/TLS inspection, how do we pass in trusted certs to the library? 

HTTPX / Requests usually allow for a path to be sent instead of a boolean as part of ssl_verify, but I don't see it in the documentation for the library. 

Thanks!

**Comment by jshcodes** [comment, marked as answer]
Hi @lostpolaris -

The dictionary will probably be treated as a string when you provide it to the `proxy` keyword, so you might need to run it through `json.loads`. I was able to make this work using `os.getenv` doing so. (Detecting an incoming string for this keyword feels like an excellent enhancement, I will create an issue for this.)

You are able to provide a filename for the `ssl_verify` keyword and it should be handed to requests when you make the API call.  Thank you for pointing out the documentation gap, I'll get a ticket created for this as well. 😄
